wp-modules/editor/tests/ModelTest.php

```diff
@@ -108,8 +108,8 @@ public function test_new_pattern_title_matches_slug() {
 		// Get the contents of the file that was saved.
 		$pattern = \PatternManager\PatternDataHandlers\get_pattern_by_name( $post->post_name );
 
-		// Make sure the post_name (aka "slug") of the post and the slug in the file match.
-		$this->assertSame( $post->post_name, $pattern['slug'] );
+		// Make sure the ->post_name is the same as the slug, other than the prefixed textdomain.
+		$this->assertStringEndsWith( $post->post_name, $pattern['slug'] );
```

Comment:
`$pattern['slug']` was `'/new-pattern-originally-created-with-pattern-manager'`

The unit tests don't have a textdomain, so `prepend_textdomain()` only prefixed `/` to it, instead of a textdomain.